main.go

```diff
@@ -69,6 +71,11 @@ func main() {
 		if err != nil {
 			return err
 		}
+		// set fields that weren't supplied to default values
```

Comment:
sorry, ik was in de veronderstelling dat de yaml lib ook naar de default tags keek.

moet dit niet voor het unmarshallen trouwens? defaults kan toch niet weten of 0 een init value was of dat het ook zo in de yaml string zat?

Reply:
Hmm zal het nog even uitproberen, maar ik had in ieder geval al gecheckt met een waarde anders dan default (pixelsize 64 ipv 16), en dan maakt het geen verschil, maar waarschijnlijk logischer om eerst de defaults te zetten inderdaad, die worden dan vanzelf overschreven indien nodig.

Reply:
ja inderdaad. 64 zal hij niet overschrijven. maar als het expliciet op 0 was gezet in de yaml (wat in dit geval overigens sowieso niet goed zou zijn), dan kan hij niet weten of dat een 0 was omdat een int 0 als init waarde heeft, of dat die 0 ook zo bedoeld was.

Reply:
klopt, heb het omgedraaid